File: help/pt_pt/exercicios_da_ficha_11/Ficha11_013/Ficha11_013.py
# Programador           -


#::::::::::::::::::::::::::::::::::::::::::::::::::
#User Defined Functions Code
#::::::::::::::::::::::::::::::::::::::::::::::::::
def linhaC(n, c):
    for i in range(0, n, 1 ):
        print(c  , end="")
    print()

#::::::::::::::::::::::::::::::::::::::::::::::::::
def trianguloC(altura, c):
    for i in range(0, altura, 1 ):
        linhaC(altura - i , c )

if __name__ == "__main__":
    altura =  int(input("Altura : "))
    c =  str(input("Caractér : "))
    trianguloC(altura , c )


#Python 3  - Automatic translated by Algorithmi 22.05
#(c) Ant0nio M@nso 	quarta, 2022 out 12 20:13

File: help/pt_pt/exercicios_da_ficha_11/Ficha11_013/Ficha11_013.js
// Programador           -

//Main Function
function main() {
    document.write("Altura : " );// print text
    altura = parseInt(prompt("Altura : " ,"0") );
    
    document.write(altura + "\n");// print input 
    document.write("Caractér : " );// print text
    c = prompt("Caractér : " ,"");
    
    document.write(c + "\n");// print input 
    trianguloC(altura , c ) ;
}

//::::::::::::::::::::::::::::::::::::::::::::::::::
//User Defined Functions Code
//::::::::::::::::::::::::::::::::::::::::::::::::::
function linhaC(n, c) {
    for( var i = 0; i < n; i = i + 1 ) {
        document.write(c );
    }
    document.write("\n" );
}
//::::::::::::::::::::::::::::::::::::::::::::::::::
function trianguloC(altura, c) {
    for( var i = 0; i < altura; i = i + 1 ) {
        linhaC(altura - i , c ) ;
    }
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	quarta, 2022 out 12 20:13
